pkg/storage/mvcc.go

```diff
@@ -265,12 +265,8 @@ func updateStatsForInline(
 }
 
 // updateStatsOnMerge updates metadata stats while merging inlined
-// values. Unfortunately, we're unable to keep accurate stats on merge
-// as the actual details of the merge play out asynchronously during
-// compaction. Instead, we undercount by adding only the size of the
-// value.Bytes byte slice (an estimated 12 bytes for timestamp,
-// included in valSize by caller). These errors are corrected during
-// splits and merges.
+// values. We only the size of the value.RawBytes byte slice (an
```

Comment:
[nit] I think we're missing a word here? That aside, I think the part about being unable to keep accurate stats on merge due to async compaction still applies. The undercounting statement was incorrect previously, we were definitely overcounting before. But I think _now_ we're actually undercounting by just a tiny bit seeing as how we never add MVCCVersionTimestampSize (even for the very merge/write). We could say as much, we still expect these errors to be corrected during splits and merges.